src/utils/arrays.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export const last = array => array[array.length - 1];
```

Comment:
I realize it's very simple, but we should add unit tests for this if we're going to add this helper. Also, this new file will need the Swift open source license preamble.